jsonenums.go

```diff
@@ -67,7 +67,9 @@ package main
 
```

Comment:
This is a bit hard to read with all of the changes in one commit, but I think this looks good.